### ISSUE TITLE
Intermittent connection issues

### ISSUE THREAD

**Issue by bhoffman0** [RunestoneInteractive/RunestoneServer#1846]
Teachers are reporting intermittent connection issues. "It let's me in for a little bit. Then when I go back to it I get the "This site can't be reached message"."

**Comment by bnmnetp** [comment]
This has got to be some kind of funny DNS thing going on.

Why do I say this?

The site called runestone.academy is a very very simple/reliable load balancer used by many thousands of sites.  I can see that it has not been down and I can see a graph of all of the incoming and outgoing traffic.  There are no gaps.

If an individual Runestone-Book-Jobe (RBJ) container were to go down the user would get a response of Gateway error not a "site can't be reached" message.  A 502 in the case of RBJ server going down, or a 504 if it was overloaded and unable to respond in a reasonable amount of time.  I had tried to set up forwarding for csawesome, and I think that may have been the problem.  I've removed that to see if this clears up these problems.

**Comment by cwaliser** [comment]
I am a teacher and having the same issues. It will let some students on and other students can't access the page. It says this site can't be reached. My tech department has done everything they can think of on our end and still hasn't fixed the issue. I am at a loss.

**Comment by bnmnetp** [comment]
I have yet to be able to reproduce an issue where I cannot connect and I have tried from several different locations with different IP addresses etc...    Again, I emphasize the the system and servers have been up 100% all day.  So I think we are just experiencing some issues with the change over in servers and the related DNS changes.  Hopefully this will settle down over the next day or two at the most.

If possible, please open the developer tools in your browser and go to the Timings tab in Safari or the Network tab in Chrome.  Then try to access the page.  If you can send me a screen shot of that, it **may** provide a clue as to what is going on.

**Comment by bnmnetp** [comment]
Also, for those really struggling with this, I have heard from a couple of people via email that using `dev.runestone.academy` works reliably for them.   I have no explanation for why that would be different than just `runestone.academy` They both point to the exact same IP address.  Which is more confirmation to me that there is DNS caching funny business going on.  But feel free to give it a try.

**Comment by bnmnetp** [comment]
SOLVED!!!

I finally figured this out.  I carefully worked my way through all of the DNS records for the runestone.academy domain, and I found a second entry that was pointing at the old IP address.  If there is a duplicate then the browser can randomly pick one of them.

I don't think I ever got unlucky and got the old one on my laptop because I was too busy checking all the time for the DNS record to expire.

So... to recap, I was right, it was a DNS problem 🎉 .  Just one of my own making 😬
